electron-app/src/server/auth.guard.ts

```diff
@@ -6,7 +6,7 @@ export class AuthGuard implements CanActivate {
   canActivate(context: ExecutionContext): boolean | Promise<boolean> | Observable<boolean> {
     const request = context.switchToHttp().getRequest();
     return (
-      request.headers.authorization === global.AUTH_ID || request.query.auth === global.AUTH_ID
+      request.headers.authorization === global.AUTH_ID || request.query.auth === global.AUTH_ID || request.path === "/misskey/display"
```

Comment:
I don't see why this change is necessary.
It looks like you pass in the auth key in the login panel.

Reply:
MissKey has a callback where the code is presented on a locally hosted page within PostyBirb, so we need to add this to allow the page to be accessed. Not automagically handled to avoid creating an X-site attack vector.